api/handler/copy.go

```diff
@@ -91,9 +96,8 @@ func (h *handler) CopyObjectHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	params := &layer.CopyObjectParams{
-		SrcBucket: srcBucket,
+		SrcObject: inf,
```

Comment:
I know u didn't choose that variable's name, but it seems like a typo or not a good name. Maybe the name become more clear if you rename it to `info`?

Reply:
Early such name already exists so I kept it. But you are right it would be better to change it.